Jellyfin.Plugin.Streamyfin/Configuration/Config.cs

```diff
@@ -30,6 +30,7 @@ public class Config
   //public Bla Wit { get; set; }
   public Search? marlinSearch { get; set; }
   public Home? home { get; set; }
+  public Settings.Settings? settings { get; set; }
```

Comment:
Not 100% sure but feel that the settings property is redundant? because the root is basically the settings root aka everything is a setting. 

```
home:
  sections:
    Trending:
      style: portrait
      type: row 
      items:
        args: 
          sortBy: AddedDate
          sortOrder: Ascending
          filterByGenre: [""Anime"", ""Comics""]";
settings:
  merlinSearch:
    enabled: true
  jellyseerrServerUrl:
    locked: true
    value: JELLYSEERR_URL_INSTANCE
```

vs

```
home:
  sections:
    Trending:
      style: portrait
      type: row 
      items:
        args: 
          sortBy: AddedDate
          sortOrder: Ascending
          filterByGenre: [""Anime"", ""Comics""]";
merlinSearch:
  enabled: true
jellyseerrServerUrl:
  locked: true
  value: JELLYSEERR_URL_INSTANCE
```